script/ci.sh

```diff
@@ -14,9 +14,10 @@ yarn test
 yarn bundle
```

Comment:
The changes in this file are because the final build which is deployed was being run with no lockfile. This meant that the versions of dependencies which end up being used in production can be different from those used during development. 

We actually found this because of an issue in the okta library, where a newer version of a dependency broke the library, this PR has more information: https://github.com/guardian/gateway/pull/2775

Reply:
Good spot, nice one